### ISSUE TITLE
Fix S2068 FN: Support utf-8 string literals

### ISSUE THREAD

**Issue by gregory-paidis-sonarsource** [SonarSource/sonar-dotnet#6394]
### Description

FN for S2068 (DoNotHardcodeCredentials).

Using C#11 utf-8 string literals, the rule is not raising an issue.

### Repro steps

```csharp
void Utf8StringLiterals()
{
    var DBConnectionString1 = "Server=localhost; Database=Test; User=SA; Password=Secret123"u8; // FN
    var DBConnectionString2 = """Server=localhost; Database=Test; User=SA; Password=Secret123"""u8; // FN
    var DBConnectionString3 = """
        Server=localhost; Database=Test; User=SA; Password=Secret123
        """u8; // FN
}
```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.
